views/components/alerts/alerts.templ

```diff
@@ -0,0 +1,63 @@
+package alerts
+
+import "soarca-gui/utils"
+
+const baseClassWarning = "flex w-full max-w-sm overflow-hidden bg-white rounded-lg shadow-md dark:bg-gray-800"
```

Comment:
Upper CammelCase for consts

Reply:
why there is no formality stating that they should be public? this is just a private const?